docs/src/content/en/docs/workflows-vnext/inngest-workflow.mdx

```diff
@@ -7,61 +7,53 @@ description: "Inngest workflow allows you to run Mastra vNext workflows with Inn
 
 [Inngest](https://www.inngest.com/docs) is a developer platform for building and running background workflows, without managing infrastructure.
 
-## Setup
-
-```sh
-npm install @mastra/inngest @mastra/core @mastra/deployer @hono/node-server
-```
+## How Inngest Works with Mastra
 
-### Local Development Environment
+Inngest provides an event delivery and orchestration layer for running Mastra vNext workflows in response to events. The integration connects Inngest's event system to Mastra's workflow engine.
 
-Inngest offers two methods for local development:
+Workflows are defined using Mastra's `createWorkflow` and `createStep` helpers, which map directly to Inngest functions through a memoized creation process. Each step of workflow is registered as a single Inngest function with a unique ID, created once and cached for efficiency. When an event occurs, Inngest invokes the corresponding function, creating a workflow run that executes steps in sequence within Inngest's context. Real-time monitoring is provided through Inngest's publish-subscribe system, where the watch method subscribes to a specific channel `workflow:${workflowId}:${runId}` for execution updates. Results are available through Inngest's API and dashboard, including status, outputs, and any errors.
```

Comment:
This is a little too much implementation detail level.

What we're looking for here, IMO, is something that really describes how the mastra-inngest interop works in terms of execution, but maybe not all the nitty-gritty detail of our implementation.

More something along the lines of:
- how does inngest function/step paradigm work
- how does mastra represesent mastra workflows as inngest workflows/steps
- how does that affect with the other workflow primitives